<launch>

  <!-- <include file="$(find mcu)/launch/mcu.launch"/> -->
  <!-- <include file="$(find odometry_laser)/launch/odometry_laser.launch"/> -->
  <!-- <include file="$(find odometry_laser)/launch/odometry_laser.launch"/> -->
  <!-- <include file="$(find kinetic)/launch/kinetic.launch"/> -->
  <node pkg="pose_robot" type="pose_robot" name="pose_robot" required="true" output = "screen" > 
  <!-- config topic odom -->
    <param name="map_frame" value="map"/>
    <param name="base_frame" value="base_link"/>
    <param name="publish_frequency" value="20"/>
    <param name="topic_pose" value="pose_robot"/>
  </node>

</launch>
